plugin-modernizer-core/src/main/resources/META-INF/rewrite/recipes.yml

```diff
@@ -442,7 +442,7 @@ recipeList:
       artifactId: plugin
       newVersion: 4.51 # See https://www.jenkins.io/blog/2022/12/14/require-java-11/
   - io.jenkins.tools.pluginmodernizer.core.recipes.UpgradeJenkinsVersion:
-      minimumVersion: 2.346.3
```

Comment:
This is NOT the minimum version. It's the last LTS supporting Java 8. This value will never changes